### ISSUE TITLE
Video RAM decoding not emulating real MZ-80K correctly

### ISSUE THREAD

**Issue by psychotimmy** [psychotimmy/picomz-80k#7]
The video RAM read and writes in picomz.c while "technically correct" do not emulate the real hardware properly. 

While VRAM reads and writes are meant to be between 0xD000 and 0xD3FF inclusive (actually 0xD3E8H as the last 24 bytes of the video RAM are not displayed), because these addresses are incompletely decoded, reads or writes from/to 0xD400-0xD7FF, 0xD800-0xDBFF and 0xDC00-0xDFFF should all have the same effect.

In other words, any read or write to an address where either address line A10 and A11 are set should have the same effect as when they do not - i.e. these all map to 0xD000 - 0xD3FF. 

(See Sharp User Club Magazine 1984, volume 4, issue 3, pages 19-20).
